tarsila/src/ui_state.rs

```diff
@@ -102,6 +104,16 @@ impl<'a> From<&'a UiState> for GuiSyncParams {
     }
 }
 
+struct Preferences {
+    default_zoom: f32,
+}
+
+impl Default for Preferences {
+    fn default() -> Self {
+        Self { default_zoom: 8. }
+    }
+}
```

Comment:
I do like the idea of a preferences object, and we will need it, but for now I think it would be better to use just a `const` on the top of the file for this config. The reason I say this is that there is a lot of configuration scattered across the program, so when we do decide to implement a preferences or settings type we need to consider also those other values. There are lots of other considerations that I might open a discussion thread or an issue for this. With the constants all on top of the files it will be relatively easy to collect them once we decide how to implement the preferences system.

Reply:
Alright, it's done. Let me know if you need any other change.